cloudinstall/core.py

```diff
@@ -575,6 +571,7 @@ def enqueue_deployed_charms(self):
                                      self.maas_state, self.config)
         self.loop.redraw_screen()
 
+    # FIXME: Use try_deploy and handle charm depends?
```

Comment:
not sure what the first part of this is about, although the second half sounds like we already did it?